### ISSUE TITLE
Adding topic wise problems 

### ISSUE THREAD

**Issue by DiligentCoder-20022001** [afrozchakure/Competitive-Programming-Solutions#34]
I guess adding topic wise problems can be helpful to beginners in the world of data structures and algorithms to recognize patterns in problems and hence easily solve them.

**Comment by afrozchakure** [comment]
Yes, I agree. I never had thought of that before starting the repository. 

It might take a bit of time from my side otherwise I'm open for pull requests from people :smile:
Feel free to raise changes if you wish. 
Again thanks for raising the issue @DiligentCoder-20022001 :tada:

**Comment by DiligentCoder-20022001** [comment]
Sure, thank you for the response @afrozchakure : )

**Comment by afrozchakure** [comment]
Let's just keep this issue open for now. So that others too know what to contribute to this repository :D
